evergreen.yml

```diff
@@ -1490,6 +1518,7 @@ tasks:
     commands:
       - func: "upload release"
       - func: "upload debug"
+      - func: "update download center feed"
```

Comment:
The `downloadcenter-update` task is being called in the `release` buildvariant, this one seems unnecessary.